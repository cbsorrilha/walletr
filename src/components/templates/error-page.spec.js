import React from 'react';
import { render } from '@testing-library/react';
import ErrorPage from './error-page';

describe('Test the error-page component', () => {
  it('should match the snapshots', () => {
    const component = render(<ErrorPage />);
    expect(component).toMatchSnapshot();
  });
});
