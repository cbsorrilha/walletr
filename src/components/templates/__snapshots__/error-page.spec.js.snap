// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the error-page component should match the snapshots 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bdVaJa fvkXj"
        direction="column"
        height="100vh"
      >
        <div
          class="sc-bwzfXH fXZdIj"
        >
          <div
            class="sc-bdVaJa eLojcs"
          >
            <div
              class="sc-bwzfXH fXZdIj"
            >
              <h1
                class="sc-dnqmqq itDPRL sc-htpNat jTSObl"
              >
                Ops!
              </h1>
              <div
                class="sc-bdVaJa bKhobn"
                direction="column"
              >
                <p
                  class="sc-htoDjs kbxgHx"
                />
                <p
                  class="sc-htoDjs kbxgHx"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bdVaJa fvkXj"
      direction="column"
      height="100vh"
    >
      <div
        class="sc-bwzfXH fXZdIj"
      >
        <div
          class="sc-bdVaJa eLojcs"
        >
          <div
            class="sc-bwzfXH fXZdIj"
          >
            <h1
              class="sc-dnqmqq itDPRL sc-htpNat jTSObl"
            >
              Ops!
            </h1>
            <div
              class="sc-bdVaJa bKhobn"
              direction="column"
            >
              <p
                class="sc-htoDjs kbxgHx"
              />
              <p
                class="sc-htoDjs kbxgHx"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
